#File
uint8[] data
uint64 data_length
uint8 extension_type
uint8 status
